resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -27,6 +27,7 @@ symbindir=${symbindir:=/opt/puppetlabs/server/bin}
 app_prefix=${app_prefix:=/opt/puppetlabs/server/apps/${real_name}}
 app_data=${app_data:=/opt/puppetlabs/server/data/${real_name}}
 app_logdir=${app_logdir:=/var/log/puppetlabs/${real_name}}
+private_config_dir=${private_config_dir:=${app_prefix}/config}
```

Comment:
@shrug @melissa @stahnma is this where you'd expect this directory to end up?  e.g. `/opt/puppetlabs/server/apps/puppetserver/config/services.d/`?  I know that some apps (razor maybe?) have like a 'share' directory underneath that path.  Easy to change it right now if you have any preferences.